### ISSUE TITLE
Cross id reenactment

### ISSUE THREAD

**Issue by ardianumam** [PeizhiYan/gaussian-dejavu#13]
Hi,

I try to do cross-id reenactment, where Obama video is used as driver (expression and pose) and other videos (Justin and x person) are used as sources to drive from. The video is [here](https://drive.google.com/file/d/1np-uhKl50J2fSmQNdzKZWWklGzsszXI6/view?usp=sharing).  I wanna make sure if I use correct parameters (which params are from driver, which params are from source). Below I list the setting that generates prev. video.
1. self.mean_uv_offsets: from source
2. self.mean_shape_coefficients: from source
3. self.mean_exp_coefficients: from driver
4. self.global_uv_delta: from source
5. self.uv_delta_blendmaps: from source
6. logits = self.mlp(exp_pose), where exp_pose = torch.cat((exp, pose), dim=1):  from driver (exp and pose)
7. self.flame(shape_params=shape, # from source
                    expression_params=batch_data['exp'][i:i+1,...], # from driver
                    pose_params=batch_data['pose'][i:i+1,...], # from driver
                    eye_pose_params=batch_data['eye_pose'][i:i+1,...]) # from driver

Could you help to confirm in case I use wrong params above?

Thank you!

**Comment by PeizhiYan** [comment]
Ah, yes. Sorry for the confusion. The ```mean_exp_coefficients``` was used in an earlier version of the code and was indeed computed from the source video, but the current version is not actually using it.

**Comment by ardianumam** [comment]
Well noted, close this one. Thanks much for the discussion.
